packages/metal/src/coreNamed.js

```diff
@@ -289,3 +289,48 @@ export function isString(val) {
  */
 export function nullFunction() {
 }
+
+/**
+ * Register a custom element for a given metal component.
+ *
+ * @param {String} tagName The tag name to use for this custom element.
+ * @param {!function()} ctor Metal component constructor.
+ * @param {?Array=} observedAttrs The attributes to be "observed".
+ * @return {void} Nothing.
+ */
+export function defineWebComponent(tagName, ctor, observedAttrs) {
```

Comment:
Actually using `metal-state` like this

```javascript
Object.keys(State.getStateStatic(Ctor));
```

Does the trick and avoids having to pass an extra argument, so that's a nice improvement.
Thanks